pkg/auth/oauth/registry/tokentimeouts.go

```diff
@@ -186,15 +196,17 @@ func (a *oauthTokenTimeoutValidator) flush(flushHorizon time.Time) {
 
 		td := a.data[tdr.name]
 		delta := a.clientTimeout(td.token.ClientName)
-		var newtimeout int32
-		if delta > 0 {
-			newtimeout = int32((td.seen.Sub(td.token.CreationTimestamp.Time) + delta) / time.Second)
-		}
-		patch := []byte(fmt.Sprintf(`[{"op": "test", "path": "/timeoutsIn", "value": %d}, {"op": "replace", "path": "/timeoutsIn", "value": %d}]`, td.token.TimeoutsIn, newtimeout))
+		newTimeout := int32((td.seen.Sub(td.token.CreationTimestamp.Time) + delta) / time.Second)
```

Comment:
this is a noop when delta is 0, see above

Reply:
Fixed.